setup.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+from setuptools import setup
+
+VERSION = '0.0.1'
+
+setup(
+    name="mondo",
+    version=VERSION,
+    description="Mondo Banking API Client",
+    author=', '.join((
+        'Tito Miguel Costa',
+        'Simon Vans-Colina <simon@simon.vc>',
```

Comment:
I took this email from your github profile but I'm wondering now whether you'd prefer I hadn't done that, regardless. Shall I remove? Any other authors I should add?